pootle/urls.py

```diff
@@ -30,7 +31,15 @@
     url(r'^accounts/evernote/', include('evernote_auth.urls')),
     url(r'^admin/reports/', include('evernote_reports.urls')),
     url(r'', include('evernote_reports.profile_urls')),
+)
+
+if settings.POOTLE_ENABLE_OFFLINE:
```

Comment:
Maybe not for including it on the URLs, but unless we are going to check on INSTALLED_APPS if the app is enabled, using a separate setting is advisable.